open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -119,19 +119,32 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        if column is None or not isinstance(column, str) or column not in df.columns or not column.strip():
</pre><p>check <code>column not in df.columns&nbsp;</code> is enough </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Done</p>